CONTRIBUTING.md

```diff
@@ -38,6 +38,26 @@ regular contributor, then you will want to set up
 do that, the CI checks will run locally before you even write your
 commit message.  This speeds up your development cycle considerably.
 
+## Installing Go ##
+
+The quickest way to get started is to install [Nix](https://nixos.org/download.html).
+Not only will this provide you with Go from within the project but an interactive
+development environment. The development environment can be started from within the
+root of the project by running the `nix develop` command.
+
+The [flake.nix](flake.nix) file within the root of this project lists the dependencies
+this project needs as well as the location of the package to build. After the
+installation of Nix, you can build the package by simply running: `nix build`
+
+This will use the Go version specified in the project and if you have a previously
+installed version of Go it will be ignored and untouched.
```

Comment:
Clarify wording:
```suggestion
Nix will use the Go version specified in the project, and if you have a previously
installed version of Go it will be ignored and untouched.
```